docs/source/test_cases/segmentation.pct.py

```diff
@@ -39,15 +48,32 @@
 
 
 # %% nbsphinx="hidden"
+def plot_one(seg, axis, colors):
```

Comment:
I think something may be going wrong with plotting the centroids here. 

The over/under segmentation centers are just stacked, but I would expect them to be in their own segments:

![image](https://github.com/user-attachments/assets/a1a45d1c-b186-4c4a-b84f-b2aee03f31dc)

And in the final example with multiple cells, we've got a centroid off in the distance :laughing::

![image](https://github.com/user-attachments/assets/e9aa9de7-af56-4517-a630-cab95941c8c2)

Reply:
This appears to be a flipping of the x and y axes :( thanks for catching